Task-50207: Html tags in tooltip.

montassarhamadi opened the pull request:
Problem: pop up displays when hovering on the document indicating the name of the document contains html tags..
Fix: No html tags appear with a document name.

houssemexo26 committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

houssemexo26 committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

houssemexo26 committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

houssemexo26 committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

hakermi committed:
MAINT-48685: Disable unneeded csrf check for some read and front actions (#1530)

Disable uneeded csrf check for some read and front actions to not end with csrf expire issue for no worthy reasons

hakermi committed:
MAINT-48685: Disable unneeded csrf check for some read and front actions (#1530)

Disable uneeded csrf check for some read and front actions to not end with csrf expire issue for no worthy reasons

hakermi committed:
MAINT-48685: Disable unneeded csrf check for some read and front actions (#1530)

Disable uneeded csrf check for some read and front actions to not end with csrf expire issue for no worthy reasons

montassarhamadi committed:
Task-50207: Html tags in tooltip.
Problem: pop up displays when hovering on the document indicating the name of the document contains html tags..
Fix: No html tags appear with a document name.

rdenarie commented on apps/portlet-documents/src/main/webapp/vue-app/files-search/components/FileSearchCard.vue:
```diff
@@ -54,10 +54,6 @@ export default {
     },
   },
   mounted() {
-    if (this.result && this.excerptTitle) {
-      // eslint-disable-next-line vue/no-mutating-props
-      this.result.title = this.excerptTitle;
-    }
```
In fact, by removing this, we lost the window.decodeURIComponent called on this.excerpts['title'][0]
So I think we should keep the if like this : 
if (this.result && this.excerptTitle) {
  // eslint-disable-next-line vue/no-mutating-props
  this.result.excerptTitle = this.excerptTitle;
}

and in ExoDocument.vue : 
v-sanitized-html="document.excerptTitle ? document.excerptTitle : document.title"

montassarhamadi committed:
excerptTitle attribute added to result object

rdenarie committed:
Merge branch 'feature/maintenance' into task-50207

rdenarie commented on apps/portlet-documents/src/main/webapp/vue-app/files-search/components/FileSearchCard.vue:
```diff
@@ -53,10 +53,11 @@ export default {
     },
   },
-  mounted() {
+  mounted()
```
formatting 
```suggestion
  mounted() {
```

hakermi committed:
MAINT-51623: Fix the mail notification sender param in the watch document notification (#1554)

ISSUE: The the sender property wasn't set in the mail notification plugin of watch documents and also doesn't take the value set in notification administrations
FIX: Set the value already set in the notification administration as  the sender value in the mail notification plugin

montassarhamadi committed:
Update apps/portlet-documents/src/main/webapp/vue-app/files-search/components/FileSearchCard.vue

Co-authored-by: Romain Dénarié <romain.denarie@exoplatform.com>

montassarhamadi committed:
Update FileSearchCard.vue

rdenarie committed:
Merge branch 'feature/maintenance' into task-50207

montassarhamadi committed:
Update FileSearchCard.vue

rdenarie commented on apps/portlet-documents/src/main/webapp/vue-app/files-search/components/FileSearchCard.vue:
```diff
@@ -53,10 +53,11 @@ export default {
     },
   },
-  mounted() {
+  mounted() 
+  { 
     if (this.result && this.excerptTitle) {
```
Formatting 
```suggestion
  mounted() { 
    if (this.result && this.excerptTitle) {
```

montassarhamadi committed:
changed accolade position

rdenarie commented on apps/portlet-documents/src/main/webapp/vue-app/files-search/components/FileSearchCard.vue:
```diff
@@ -53,10 +53,10 @@ export default {
     },
   },
-  mounted() {
+  mounted() { 
```
```suggestion
  mounted() {
```

rdenarie committed:
Merge branch 'feature/maintenance' into task-50207

rdenarie commented on apps/portlet-documents/src/main/webapp/vue-app/files-search/components/FileSearchCard.vue:
```diff
@@ -53,10 +53,10 @@ export default {
     if (this.result && this.excerptTitle) {
-      // eslint-disable-next-line vue/no-mutating-props
-      this.result.title = this.excerptTitle;
+    // eslint-disable-next-line vue/no-mutating-props
```
```suggestion
      // eslint-disable-next-line vue/no-mutating-props
```

montassarhamadi committed:
Update apps/portlet-documents/src/main/webapp/vue-app/files-search/components/FileSearchCard.vue

Co-authored-by: Romain Dénarié <romain.denarie@exoplatform.com>

rdenarie committed:
Merge branch 'feature/maintenance' into task-50207

rdenarie reviewed (changes requested):
There is still a modification to do

montassarhamadi committed:
Update apps/portlet-documents/src/main/webapp/vue-app/files-search/components/FileSearchCard.vue

Co-authored-by: Romain Dénarié <romain.denarie@exoplatform.com>
